services/src/test-library/src/main/java/org/openkilda/testing/service/database/DatabaseNeoImpl.java

```diff
@@ -59,7 +61,11 @@ public DatabaseNeoImpl(@org.springframework.beans.factory.annotation.Value("${ne
     @Override
     public void destroy() {
         if (neo != null) {
-            neo.close();
+            try {
+                neo.close();
+            } catch (Exception e) {
```

Comment:
Why do we need this code? I don't understand why we should catch exceptions in test code. Did you face any issues here?

Reply:
I/O exceptions in destructors are not good. Decided to avoid it anyway.

Reply:
Removed the code.